content/docs/user-guide/gems/reference/debug/imgui.md

```diff
@@ -1,8 +1,102 @@
----
-linkTitle: Immediate Mode GUI (IMGUI)
-title: Immediate Mode GUI (IMGUI) Gem
-description: The Immediate Mode GUI (IMGUI) Gem provides the 3rdParty library IMGUI which can be used to create runtime immediate mode overlays for debugging and profiling in Open 3D Engine (O3DE) projects.
-toc: true
----
-
-The Immediate Mode GUI (IMGUI) Gem provides the 3rdParty library IMGUI which can be used to create run time immediate mode overlays for debugging and profiling information in Open 3D Engine. Atom ImGUI is used by the PhysX debugging tools.
+---
+linkTitle: Immediate Mode GUI (IMGUI)
+title: Immediate Mode GUI (IMGUI) Gem
+description: The Immediate Mode GUI (IMGUI) Gem provides the 3rdParty library IMGUI which can be used to create runtime immediate mode overlays for debugging and profiling in Open 3D Engine (O3DE) projects.
+toc: true
+---
+
+The Immediate Mode GUI (ImGui) Gem provides the 3rdParty library [ImGui](https://github.com/ocornut/imgui), which can be used to create run time immediate mode menus and overlays for debugging and profiling information in Open 3D Engine. You can use it to access existing O3DE tooling or set up your own ImGui menus for your components or other game code.
```

Comment:
nit: Here we should probably refer to `ImGui` as `Dear ImGui` which is the full name of the library

Reply:
Sure.